UnityGLTF/ASsets/UnityGLTF/Scripts/UniformMaps.meta

```diff
@@ -1,7 +1,7 @@
 fileFormatVersion: 2
-guid: b8468e126fb62374b8675d3125eb2eec
+guid: e766804828a966240ab9d312c8abb342
```

Comment:
why did the meta's change?

Reply:
I'm more curious why there is an old version at all, tbh. I make that folder in this branch.